tests/test-notebook/src/actions.spec.ts

```diff
@@ -912,79 +912,79 @@ describe('@jupyterlab/notebook', () => {
     describe('#extendSelectionAbove()', () => {
       it('should extend the selection to the cell above', () => {
         widget.activeCellIndex = 1;
-        NotebookActions.extendSelectionAbove(widget);
+        NotebookActions.extendSelectionAbove(widget, false);
```

Comment:
With the default being false, now we don't need to change any of the existing calls in this file.